.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+node_js:
+  - '6'
+  - '8'
+before_script:
+  - yarn run dist-test
+script:
+  - yarn lint
+  - yarn run test-es6
+  - yarn run test-dist
+  - yarn run test-parity
+  - yarn run prettier && git diff --exit-code
```

Comment:
Will this fail if the built library isn't updated?

Reply:
This is similar to the `clang-format` check we have in `h3`. It runs a formatter over the `lib`, `test`, and `build` directories (slightly confusingly, `build` is code required for the emscripten build, not the built files, which are in `dist`) and fails if any files were changed during formatting.